apps/cfp/views.py

```diff
@@ -827,7 +827,7 @@ def proposal_messages(proposal_id):
             if channel := app.config.get("CONTENT_IRC_CHANNEL"):
                 # WARNING: don't send personal information via this (the channel is public)
                 irc_send(
-                    f"{channel} New CfP message for {proposal.human_type}: {external_url('cfp_review.message_proposer', proposal_id=proposal_id)}"
+                    f"✉️ {channel} New CfP message for {proposal.human_type}: {external_url('cfp_review.message_proposer', proposal_id=proposal_id)} ✉️"
```

Comment:
the first ✉️ should be the other side of {channel} or it'll be changing where it tries to send rather than added to the message

Reply:
Good catch, I've made `channel` an explicit argument in `irc_send` to make this clearer